test/test_check_health.rb

```diff
@@ -0,0 +1,95 @@
+        'You have no significant injuries.'
+    ]
+    $history = $server_buffer.dup
+    assert_healthy(DRCH.check_health)
+  end
+
+  def test_that_fatigue_line_does_not_interfere
+    $server_buffer = [
+        'Your body feels at full strength.',
+        'You are slightly fatigued.',
+        'Your spirit feels full of life.',
+        'You have no significant injuries.'
+    ]
+    $history = $server_buffer.dup
+    assert_healthy(DRCH.check_health)
+  end
+
+  def test_that_poisoned_person_is_poisoned
+    $server_buffer = [
+        'Your body feels at full strength.',
+        'Your spirit feels full of life.',
+        'You have some tiny scratches to the neck, minor swelling and bruising around the left arm compounded by cuts and bruises about the left arm, minor swelling and bruising around the right leg compounded by cuts and bruises about the right leg, minor swelling and bruising around the left leg compounded by cuts and bruises about the left leg, some minor abrasions to the left hand, some tiny scratches to the chest, some tiny scratches to the abdomen, some tiny scratches to the back, some minor abrasions to the right eye.',
+        'You have a mildly poisoned right leg.'
+    ]
+    $history = $server_buffer.dup
+    health = DRCH.check_health
+    assert_empty(health['parasites'])
+    assert_equal(health['diseased'], false)
+    assert_equal(health['poisoned'],true)
+    assert_equal(health['wounds'].empty?, false)
+  end
+
+  def test_that_wounded_person_is_healthy_after_healing
+    $server_buffer = [
+        'Your body feels at full strength.',
+        'Your spirit feels full of life.',
+        'You have some tiny scratches to the neck, minor swelling and bruising around the left arm compounded by cuts and bruises about the left arm, minor swelling and bruising around the right leg compounded by cuts and bruises about the right leg, minor swelling and bruising around the left leg compounded by cuts and bruises about the left leg, some minor abrasions to the left hand, some tiny scratches to the chest, some tiny scratches to the abdomen, some tiny scratches to the back, some minor abrasions to the right eye.',
+        'You have no significant injuries.',
+        'Empath touches you.',
+        'You feel a warmth radiate from Empath\'s touch.',
+        'You have a brief sensation that leaves your wounds tingling.',
+        'Your external left leg and chest wounds feel fully healed.',
+        'Your internal left leg and chest wounds feel fully healed.',
+        'Your external left leg and chest scars feel fully healed.',
+        'Your internal left leg and chest scars feel fully healed.',
+        'Empath whispers, "Done!"',
+        'Your body feels at full strength.',
+        'Your spirit feels full of life.',
+        'You have no significant injuries.'
+    ]
+    $history = $server_buffer.dup
+    assert_healthy(DRCH.check_health)
+  end
+
+  def test_that_diseased_person_is_diseased
```

Comment:
This test will fail. There is something up with these particular messages that are throwing off check_health. Yay testing :)